tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -150,6 +150,16 @@ private[scalanative] object LLVM {
     copyOutput(config, buildPath)
   }
 
+  def dsymutil(config: Config, path: Path) = {
+    val dsymutil = Discover.discover("dsymutil", "LLVM_BIN")
+    val command = Seq(dsymutil.abs, path.abs)
+    val proc = Process(command, config.workDir.toFile())
+    val result = proc ! Logger.toProcessLogger(config.logger)
+    if (result != 0) {
+      throw new BuildException(s"Failed to run dsymutil ${path}")
+    }
```

Comment:
If the `dsymutil` fails it should rather be a warning saying that we failed to postprocess binary and the stack traces would not work correctly.